_1327/templates/base.html

```diff
@@ -159,6 +159,15 @@
 				</ul>
 			</div>
 			<div class="login-area pull-right hidden-print">
+				{% get_current_language as LANGUAGE_CODE %}
+				{% if LANGUAGE_CODE == "de" %}
+					<span class="selected-language">DE</span> |
+					<a onclick="setLanguage('en');" href="#">EN</a>
+				{% else %}
+					<a onclick="setLanguage('de');" href="#">DE</a> |
```

Comment:
in evap we currently have this PR open: https://github.com/fsr-itse/EvaP/pull/1266 don't know whether it applies here as well.